docs/core/seeding.md

```diff
@@ -0,0 +1,240 @@
+
+\begin{equation*}
+\frac{A^2+1}{B^2} > (2 R^{min})^2 = \left ( \frac{2 \cdot p_T^{min}}{300 \cdot B_z} \right)^2 \equiv \textnormal{minHelixDiameter2} ,
+\end{equation*}
+
+where $B_z$ is the magnetic field.
+
+(r-zCoordinates)=
+:::{figure} ../figures/seeding/r-zCoordinates.svg
+:width: 500px
+:align: center
+The r-z projection of the detector with the same charged particle track. The track is depicted with the same colours as in the previous figure.
+:::
+       
+The track is not an ideal helix. At each detector layer (or any other material)
+scattering may occur making the helix approximate. 
+The algorithm will check if the triplet forms a nearly straight line
+in the $r/z$ plane (see {numref}`r-zCoordinates`) as the particle path in the $r/z$ plane is
+unaffected by the magnetic field. This is split into two parts; the first test occurs before the calculation of the helix
+circle. Therefore, the deviation from a straight line is compared to the
+maximum allowed scattering at minimum $p_T$ scaled by the forward angle:
+
+\begin{equation*}
+\left (\cot \theta_b - \cot \theta_t \right )^2 < \sigma^2_{p_T^{min}} + \sigma_f^2,
+\end{equation*}
+
+This check takes into account the squared uncertainty in the difference between slopes ($\sigma_f^2$) and the
+scattering term `scatteringInRegion2`  ($\sigma^2_{p_T^{min}}$), which is calculated from
+`sigmaScattering`, the configurable number of sigmas of scattering angle
+to be considered, and `maxScatteringAngle2`, which is evaluated from the
+Lynch & Dahl correction[^Tanabashi:2018]$^,$[^Lynch:1991] of the Highland equation assuming the lowest
+allowed $p_T$:
+
+\begin{equation*}
+\Theta_0^{min} = \frac{13.6 \text{MeV}}{p_T^{min}} \cdot \sqrt{\frac{z_q^2 L}{\beta^2 L_0}} \left(1+0.038 \ln \left(\frac{z_q^2 L}{\beta^2 L_0}\right) \right),
+\end{equation*}
+
+The second part check against the multiple scattering term `p2scatterSigma` ($\sigma^2_{p_T^{estimated}}$) assuming the seed $p_T$ estimation, instead of the minimum allowed $p_T$.
+This term is inversely proportional to the momentum and accounts for the curvature of the seed; smaller scattering angle is permitted for higher momentum.
+
+\begin{equation*}
+\left (\cot \theta_b - \cot \theta_t \right ) ^2 < \sigma^2_{p_T^{estimated}} + \sigma_f^2,
+\end{equation*}
+
+The last cut applied in this function is on the transverse impact parameter (or DCA -
+distance of closest approach), which is the distance of the perigee of a track from
+the interaction region in $mm$ of detector radius. It is calculated and cut on
+before storing all top SP compatible with both the current middle SP and current
+bottom SP. 
+
+(impactParameter)=
+:::{figure} ../figures/seeding/impactParameter.svg
+:width: 400px
+:align: center
+oi
+:::
+
+Assuming the middle layer SP in the origin of the $x/y$ frame, as in {numref}`impactParameter`. 
+The distance between the center of the helix and the interaction point (IP) is given by
```

Comment:
```suggestion
The distance between the centre of the helix and the interaction point (IP) is given by
```
Since we are using Br.Eng. and to match l.78